### PR TITLE
Promote did_you_mean to default gem

### PR DESCRIPTION
**Pull request by yuki24**
This is a continuation of https://github.com/ruby/ruby/pull/2631. Let's see what the CI says about it.

### CODE REVIEW

File: lib/did_you_mean/levenshtein.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
+module DidYouMean
+  module Levenshtein # :nodoc:
```
Comment (simi) [inline lib/did_you_mean/levenshtein.rb]: What about to move levenshtein to lib/levenshtien.rb and make it proper standard library item?

**Comment by yuki24** [comment]
Alright, all the builds are green! I'll go ahead and merge this pull request. Thanks to @kddeisz for all your help on this! Really appreciated 🚀

File: lib/did_you_mean/levenshtein.rb (hunk 1 above)
Comment (yuki24) [reply lib/did_you_mean/levenshtein.rb]: Sounds like something we can iterate on. Would appreciate if you could open an issue on https://bugs.ruby-lang.org/projects/ruby-trunk/issues.

Comment (simi) [reply lib/did_you_mean/levenshtein.rb]: yup, I'll open issue in there covering this idea

**Pull request merged**